docs/components/Listbox/WithSearch.vue

```diff
@@ -192,14 +192,14 @@ const handleFilterSegments = (query: string) => {
   filteredSegments.value = nextFilteredSegments;
 };
 
-const handleQueryChange = (e: Event, value: string) => {
+const handleQueryChange = (value: string) => {
```

Comment:
Or another way is add `_` as prefix of unused params. 
e.g: `_e`

Reply:
e is unused in function